src/v/utils/unresolved_address.cc

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
why is this in utils and not in `rpc::` namespace ? 

what does util have to do with network resolution?